packages/openapi3/src/openapi.ts

```diff
@@ -1307,7 +1307,7 @@ function createOAPIEmitter(program: Program, options: ResolvedOpenAPI3EmitterOpt
     const values = [];
     if (e.members.size === 0) {
       reportUnsupportedUnion("empty");
-      return undefined;
+      return {};
```

Comment:
I definitely don't want to change this to a warning because it would require making various error scenarios "kinda work but not really". 

Originally this PR was just going to be fixing the typo that referenced OpenAPI 2 instead of 3.

Reply:
that's fair, maybe just create a separate issue then and keep it like that for now